definitions/divorce/json/CaseEventToFields.json

```diff
@@ -3317,45 +3461,187 @@
     "PageLabel": "Statement of case - separation",
     "PageDisplayOrder": 13,
     "PageShowCondition": "D8ReasonForDivorce=\"separation-*\"",
-    "ShowSummaryChangeOption": "No"
+    "ShowSummaryChangeOption": "No",
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/calculate-separation-fields"
   },
   {
-    "LiveFrom": "01/01/2017",
+    "LiveFrom": "05/06/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solicitorUpdate",
-    "CaseFieldID": "SeparationLivedTogetherAsCoupleAgain",
+    "CaseFieldID": "LabSolSOCSeparation-Para2",
     "PageFieldDisplayOrder": 2,
-    "DisplayContext": "MANDATORY",
+    "DisplayContext": "READONLY",
     "PageID": "SolSOCSeparationUpdate",
     "PageLabel": "Statement of case - separation",
     "PageDisplayOrder": 13,
-    "ShowSummaryChangeOption": "Yes"
+    "ShowSummaryChangeOption": "No"
   },
   {
-    "LiveFrom": "01/01/2017",
+    "LiveFrom": "05/06/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solicitorUpdate",
-    "CaseFieldID": "SeparationLivedTogetherAsCoupleAgainDetails",
+    "CaseFieldID": "D8MentalSeparationDate",
     "PageFieldDisplayOrder": 3,
     "DisplayContext": "MANDATORY",
     "PageID": "SolSOCSeparationUpdate",
     "PageLabel": "Statement of case - separation",
     "PageDisplayOrder": 13,
-    "FieldShowCondition": "SeparationLivedTogetherAsCoupleAgain=\"Yes\"",
     "ShowSummaryChangeOption": "Yes"
   },
   {
-    "LiveFrom": "01/01/2017",
+    "LiveFrom": "05/06/2019",
     "CaseTypeID": "DIVORCE",
     "CaseEventID": "solicitorUpdate",
-    "CaseFieldID": "D8ReasonForDivorceSeperationDate",
+    "CaseFieldID": "D8PhysicalSeparationDate",
     "PageFieldDisplayOrder": 4,
     "DisplayContext": "MANDATORY",
     "PageID": "SolSOCSeparationUpdate",
     "PageLabel": "Statement of case - separation",
     "PageDisplayOrder": 13,
     "ShowSummaryChangeOption": "Yes"
   },
+  {
+    "LiveFrom": "05/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "solicitorUpdate",
+    "CaseFieldID": "D8SeparationReferenceDate",
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": "SolSOCLivedApartUpdate",
+    "PageLabel": "Lived Apart for the entire time",
+    "PageDisplayOrder": 14,
+    "PageShowCondition": "D8ReasonForDivorce=\"separation-*\"",
```

Comment:
The pageshow condition will be updated to D8ReasonForDivorce="separation-*" OR D8ReasonForDivorce="desertion" once RDM-2804 is resolved